docs/contributing/contributing-rules.md

```diff
@@ -493,4 +493,15 @@ References help provide more context to a developer on what the issue is, and ho
       - https://sequelize.org/docs/v6/core-concepts/raw-queries/#replacements
     ```
 
+## Updating existing rules
+
+To update an existing open-source rule, follow these steps:
+
+1. Find a rule you want to update in the [semgrep-rules](https://github.com/returntocorp/semgrep-rules/) repository.
+2. Submit a PR to the repository with your new update.
+3. Follow the same instructions and recommendations as you can find in the rest of this document. For example the security category has specific metadata requirements.
+4. Leave a message in the PR. Include what have you changed and why. What is the motivation for your update? TODO - add an example of a nicely written PR.
```

Comment:
@inkz @LewisArdern do you have an example of a great PR to our repository with a great message? We can share an old merged PR example here.

Reply:
This looks good to me otherwise!

Reply:
same! lgtm :+1:

Reply:
```suggestion
4. Leave a message in the PR. Explain why are you making changes. What is the motivation for this rule update?
```